### DISCUSSION TITLE
Pages

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#37903] [Other Feature Feedback, Questions, & Ideas]
What are pages?

**Comment by jfullstackdev** [comment]
GitHub Pages is an added feature of GitHub, originally intend to host solely the documentation of a project, that's why it's only static web page

other users are using that for their landing page with a free and simple setup,

check this out 

https://github.com/jdevstatic/github-pages-tutorial

if you are asking the category Pages here on GitHub Community, questions or discussions will be all about GitHub Pages

![image](https://user-images.githubusercontent.com/47092464/199152562-8a25fa23-b53e-46f8-9a63-d8ab4bf7ba18.png)
